airbyte-integrations/connectors/source-hubplanner/unit_tests/test_source.py

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+from unittest.mock import MagicMock
+
+from source_hubplanner.source import SourceHubplanner
+
+
+def test_check_connection(mocker):
+    source = SourceHubplanner()
+    logger_mock, config_mock = MagicMock(), MagicMock()
+    assert source.check_connection(logger_mock, config_mock) == (True, None)
```

Comment:
@marcosmarxm for example, I am a little confused how I am supposed to set this up to pass. All the other connectors look like they are setting up fixtures that just have like `"api_key": "test_api_key"` as their config. I don't understand why something like that would pass without a valid config.

Reply:
The goal of this test is to assert the behavior of check_connection on successful or failed connection response by the API. You should use `request_mock` to forge mock success and failure request and assert on the behavior of `check_connection`.

Reply:
@alafanechere so I would copy basically an example of a "failed response" from the hubplanner API, set up a test to mock that as the response, and make sure that `check_connection` fails in that instance?